docs/advanced/api/api.rst

```diff
@@ -32,6 +32,9 @@ To start the agents on a node, (where ``${CONTROL_NODE}`` is the address of the
 .. task:: enable_flocker_agent ${NODE_NAME} ${CONTROL_NODE}
    :prompt: [root@agent-node]#
 
+You will need to provide API end users with a certificate for authentication before they can use the API.
+Please see the :doc:`API authentication guide <./authentication>` for more information.
+
```

Comment:
This is not Fedora and CentOS specific. Perhaps put this below the API Details heading, and direct it more to API users. "You will need a certificate for authentication before you can use the API."